ur_description/urdf/ur.xacro

```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://wiki.ros.org/xacro" name="ur">
-
-   <!-- import main macro -->
+   <!-- Import main macro -->
    <xacro:include filename="$(find ur_description)/urdf/inc/ur_macro.xacro"/>
 
-   <!-- parameters -->
-   <xacro:arg name="joint_limit_params" default=""/>
-   <xacro:arg name="kinematics_params" default=""/>
-   <xacro:arg name="physical_params" default=""/>
-   <xacro:arg name="visual_params" default=""/>
-   <!-- legal values:
+   <!-- Parameters -->
+   <xacro:arg name="model" default="5"/>
```

Comment:
```suggestion
   <xacro:arg name="robot_model" default="ur5"/>
```
As we nowadays do add the `robot_model` argument to the robot's name, we should call the argument the same here and add the `ur` prefix to its possible values. Also, possible values should probably be added as a comment above.